### DISCUSSION TITLE
Add an option to download all the "versions" of a LoRa

### DISCUSSION THREAD

**Discussion by CesarCruzUtec** [BlafKing/sd-civitai-browser-plus#296] [Ideas]
There are some LoRa models in civitai that have many versions, for example this one: https://civitai.com/models/352581
![image](https://github.com/BlafKing/sd-civitai-browser-plus/assets/67798431/2879dc65-896a-4deb-9eb6-ff4d4fcc66ad)
To download all versions, one must access each individually, which is not challenging, but the page reloads with every version change. Since LoRa models download quickly, the page refreshes upon completion, making it difficult to queue downloads. A button enabling the download of all versions simultaneously, similar to the existing feature for downloading all models on a page, would be highly beneficial.

**Comment by Pemmican6** [comment]
This would be an amazing addition, to be able to iterate through all the versions and download them would be huge.
An added dream feature would be to check off any number of them and download just those :)
